cmd/kubeadm/app/cmd/reset.go

```diff
@@ -182,6 +182,18 @@ func (r *Reset) Run(out io.Writer, client clientset.Interface) error {
 	}
 	resetConfigDir(kubeadmconstants.KubernetesDir, r.certsDir)
 
+	// Output help text instructing user how to remove iptables rules
+	msg := `
+	The reset process does not reset or clean up iptables rules or IPVS tables.
+	If you wish to reset iptables, you must do so manually.
+	For example: 
+	iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
+
+	If your cluster was setup to utilize IPVS, run ipvsadm --clear (or similar)
+	to reset your system's IPVS tables.
+	`
+	fmt.Print(msg)
```

Comment:
i think we should print this in the cobra commands help screen e.g. `kubeadm reset --help` instead of printing it on every run.

will i will defer to @timothysc 's decision on this.

Reply:
It only matters when they run the command, so I'm fine with it here.  Otherwise it would just be noise in the help window.